src/Haskell/Ide/Engine/Plugin/HieExtras.hs

```diff
@@ -261,8 +286,8 @@ instance ModuleCache CachedCompletions where
       , importableModules = moduleNames
       }
 
-getCompletions :: Uri -> PosPrefixInfo -> IdeDeferM (IdeResult [J.CompletionItem])
-getCompletions uri prefixInfo = pluginGetFile "getCompletions: " uri $ \file -> do
+getCompletions :: Uri -> PosPrefixInfo -> Bool -> IdeDeferM (IdeResult [J.CompletionItem])
```

Comment:
Can we wrap that `Bool` in a newtype? Give that we don't have much docs it would be useful to read that the Bool is for

Reply:
Would some haddock documentation suffice instead? The setting is read in as a bool from JSON so it would be nice to not have to convert it